redbot/core/commands/context.py

```diff
@@ -86,6 +86,24 @@ def __init__(self, **attrs):
         else:
             return True
 
+    async def react_quietly(
+        self, reaction: Union[discord.Emoji, discord.Reaction, discord.PartialEmoji, str]
+    ) -> bool:
+        """Adds a reaction to to the command message.
+        Returns
+        -------
+        bool
+            :code:`True` if adding the reaction succeeded.
+        """
+        try:
+            await self.message.add_reaction(reaction)
+        except (discord.HTTPException, discord.NotFound, discord.Forbidden):
```

Comment:
The other two here inherit from this one
```suggestion
        except discord.HTTPException:
```